### PR TITLE
Skeleton README with development instructions

### PR DESCRIPTION
**Pull request by holly-cummins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,45 @@
...
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.com/docs/gatsby-starters/)._
+# Local development 
 
-## 🚀 Quick start
+Local development is a Quarkus-like experience, with continuous testing and live reload. 
+You will need node and npm set. 
 
-1.  **Create a Gatsby site.**
+```
+brew install nvm
```
Comment (maxandersen) [inline README.md:11]: this is not enough.

Need to read the brew output to then setup .zshrc for it to work.

```
  export NVM_DIR="$HOME/.nvm"
  [ -s "/opt/homebrew/opt/nvm/nvm.sh" ] && \. "/opt/homebrew/opt/nvm/nvm.sh"  # This loads nvm
  [ -s "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm" ] && \. "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm"  # This loads nvm bash_completion
  ```

File: README.md [hunk 2]
```diff
@@ -1,105 +1,55 @@
...
-    ```
+```
+nvm install 14
+nvm use 14
+```
 
-1.  **Start developing.**
+In one terminal, run tests
+```
+npm run test:watch
```
Comment (maxandersen) [inline README.md:34]: this gives me:

```
npm run test:watch

> quarkus-extensions@ test:watch /Users/manderse/code/quarkusio/extensions.quarkus.io
> ACTIVE_ENV=test SUPPRESS_ENV_OUTPUT=true jest --watch --testPathIgnorePatterns test-integration/

sh: jest: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! quarkus-extensions@ test:watch: `ACTIVE_ENV=test SUPPRESS_ENV_OUTPUT=true jest --watch --testPathIgnorePatterns test-integration/`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the quarkus-extensions@ test:watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/manderse/.npm/_logs/2022-11-16T10_12_04_227Z-debug.log
```

File: README.md [hunk 3]
```diff
@@ -1,105 +1,62 @@
...
-    ```
+In one terminal, run tests
+```
+npm run test:watch
+```
 
-1.  **Open the source code and start editing!**
+In another terminal, run the site
+```
+gatsby develop
```
Comment (maxandersen) [inline README.md:39]: this now gives me:

```
gatsby develop

 ERROR

Gatsby requires Node.js 18.0.0 or higher (you have v14.21.1).
Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
```

Comment (maxandersen) [reply README.md:39]:
```
nvm install 18 
nvm use 18
gatsby develop
```

and it works.

File: README.md (hunk 2 above)
Comment (holly-cummins) [reply README.md:35]: Forgot the `npm install`. Pushed update.

File: README.md (hunk 1 above)
Comment (holly-cummins) [reply README.md:12]: Updated, thanks!

**Pull request merged**